src/content/app/species/components/species-title-area/SpeciesTitleArea.module.css

```diff
@@ -40,6 +40,26 @@
   line-height: 1;
 }
 
+.assemblyCountWrapper {
+  display: inline-flex;
+  grid-area: assembly-count;
+  column-gap: 8px;
+}
```

Comment:
You probably want `align-items: center` on this element

Reply:
I already tried this and it didn't work. However, switched to using `justify-content: space-around`.

Reply:
No, no, that's not what I meant. I didn't mean instead of justify-content. I meant that you probably want to vertically align the text and the lozenge as well, with align-items.

Reply:
I used `align-items: center` and it reduces the vertical size of the info pill:

![Screenshot 2024-02-07 at 12 12 04](https://github.com/Ensembl/ensembl-client/assets/32512909/73ad3874-c129-4ff5-8be1-a0c8bc317688)